### PR TITLE
Refine.

### PR DESCRIPTION
**Pull request by DreamXzxy**

### CODE REVIEW

File: test/FlashLoanV1Router01.spec.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { solidity, MockProvider, createFixtureLoader } from 'ethereum-waffle'
 import { ecsign } from 'ethereumjs-util'
 
 import { expandTo18Decimals, getApprovalDigest, mineBlock, MINIMUM_LIQUIDITY } from './shared/utilities'
-import { v2Fixture } from './shared/fixtures'
+import { Router01Fixture } from './shared/fixtures'
```
Comment (TT1943) [inline test/FlashLoanV1Router01.spec.ts]: naming style is wrong, either use 'router01Fixture' or 'v1Fixture'

File: test/shared/fixtures.ts [hunk 2]
```diff
@@ -26,11 +26,10 @@ interface V2Fixture {
   receiver: Contract
 }
 
-export async function v2Fixture(provider: Web3Provider, [wallet]: Wallet[]): Promise<V2Fixture> {
+export async function Router01Fixture(provider: Web3Provider, [wallet]: Wallet[]): Promise<Router01Fixture> {
```
Comment (TT1943) [inline test/shared/fixtures.ts]: naming

File: test/FlashLoanV1Router01.spec.ts (hunk 1 above)
Comment (DreamXzxy) [reply test/FlashLoanV1Router01.spec.ts]: Done.

File: test/shared/fixtures.ts (hunk 2 above)
Comment (DreamXzxy) [reply test/shared/fixtures.ts]: Done.

**Pull request merged**